### ISSUE TITLE
Character Encoding on Tweet Link

### ISSUE THREAD

**Issue by jedkirby** [jedkirby/portfolio#38]
When using the "Tweet" link on any of the blog post articles, it encodes the title of the post for use within the tweet. This is encoding special characters, like `&` to be `&amp;` for example, which doesn't look right on the _actual_ tweet, see:

<img width="604" alt="screen shot 2017-06-26 at 08 57 15" src="https://user-images.githubusercontent.com/1528691/27529754-a0f4a584-5a4d-11e7-9e79-52b2b5fdd0aa.png">

Basically, see if this can be fixed.
